src/pykeen/losses.py

```diff
@@ -284,13 +295,39 @@ def __init__(self, reduction: str = "mean"):
         super().__init__(reduction=reduction)
         self._reduction_method = _REDUCTION_METHODS[reduction]
 
+    # TODO: mypy does not seem to like this annotation
```

Comment:
double checking, it looks like torch.Module does some sketchy thing with defining forward as an instance variable to get around mypy checking. We might not be able to address this so easily, so don't worry about making it work in this PR

Reply:
For reference, this is the definition: https://github.com/pytorch/pytorch/blob/1eba9b3aa3c43f86f4a2c807ac8e12c4a7767340/torch/nn/modules/module.py#L383